src/fides/common/api/v1/urn_registry.py

```diff
@@ -112,6 +112,9 @@
     "/privacy-request/transfer/{privacy_request_id}/{rule_key}"
 )
 
+# Purpose URLs
+PURPOSES = "/purposes"
```

Comment:
Not sure if it's best to make this its own URL or if it should go under an existing path